static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +53,41 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically pushed to `less` command line tool if
+`less` is runnable (installed and can be executed in your shell). To be exact,
+the actual command is `less --chop-long-lines --clear-screen`.
+
+If `less` is not runnable (usually Microsoft Windows), the ouput is simply
+printed out.
+
+> Note Microsoft Windows users may also want to read
+> [this how-to](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
+
+### Providing a custom pager
+
+You can also override the default pager via `DVC_PAGER` environemnt variable.
+
+For example:
+
+Following command will replace the default pager with
+[`more`](<https://en.wikipedia.org/wiki/More_(command)>) for a single call
+
+```bash
+$ DVC_PAGER=more dvc pipeline show --ascii my-pipeline.dvc
+```
+
+To make it persistent, you need to add it to a shell depending file.
+
+For `Bash` it means adding this line
+
+```bash
+export DVC_PAGER=more
+```
+
+to `~/.bashrc`.
```

Comment:
For a persistent change, define `DVC_PAGER` in the shell configuration. For example in Bash, we could add the following line to `~/.bashrc`:

```bash
export DVC_PAGER=more
```